src/main/java/gift/product/dto/ProductRequest.java

```diff
@@ -1,4 +1,16 @@
 package gift.product.dto;
 
-public record ProductRequest(String name, Long price, String imageUrl) {
+import jakarta.validation.constraints.*;
+
+public record ProductRequest(
+        @NotBlank(message = "상품 이름은 비워둘 수 없습니다.")
+        String name,
+
+        @NotNull(message = "가격은 비워둘 수 없습니다.")
+        @Min(value = 1, message = "가격은 1원 이상이어야 합니다.")
+        @Max(value = 200000000, message = "가격은 2억원 이하이어야 합니다.")
```

Comment:
사소할 수 있지만 3자릿수마다 `_`를 넣어준다면 조금 더 한눈에 숫자를 파악할 수 있을 것 같아요. 😃 

```suggestion
        @Max(value = 200_000_000, message = "가격은 2억원 이하이어야 합니다.")
```